Newsletter 13, avril (#64)
InseeFrLab/ssphub: content/infolettre/infolettre_13/index.md

Après `ChatGPT`, le navigateur `Bing` s'enrichit également d'un [créateur d'image](https://www.bing.com/create) basé sur `Dall-E`. 
Pendant ce temps, `Midjourney` (une des principales alternatives de création d'images) sort sa [V5](https://www.lesnumeriques.com/intelligence-artificielle/midjourney-v5-est-disponible-comment-l-utiliser-n208168.html), améliorant encore le réalisme du rendu. 

Face à la recrudescence des montages utilisant `Midjourney`, qu'il s'agisse de _deep fakes_ ou
d'images à vocation humoristique, _Le Monde_ a publié un
guide pour [reconnaître une image générée par une IA](https://www.lemonde.fr/les-decodeurs/article/2023/03/30/comment-reconnaitre-une-image-generee-par-midjourney_6167525_4355770.html#xtor=AL-32280270-%5Bdefault%5D-%5Bios%5D).

![](https://preview.redd.it/pope-francis-goes-to-space-v0-7e9161rhoypa1.png?width=640&crop=smart&auto=webp&s=75d424b86993ca3b081e4182e774b51b4b6d3bce)
_Source: [Reddit](https://www.reddit.com/r/midjourney/comments/1221l3j/pope_francis_goes_to_space/)_

> La publication de modèles de langage, permettant une grande variété d'applications comme les _chatbots_ ou l'extraction d'information, continue à suivre un rythme presque quotidien. 

Author: ```suggestion
Alors que `ChatGPT` est déjà utilisé par plus de 100 millions d'utilisateurs, seulement quatre mois après sa sortie, il est légitime de se poser la question, comme le fait [Gaspard Koenig dans _Les Echos_](https://www.linkedin.com/posts/gaspard-koenig-36b49647_je-persiste-et-signe-in-les-echos-du-jour-activity-7049265627668078592-Nzm5?utm_source=share&utm_medium=member_desktop) du rapport à la vérité des IA génératives et des implications sociétales de la généralisation de ces assistants.

Les prospectivistes, après s'être consacrés au _bitcoin_ et à la _blockchain_, comme [Jean Detrez - le héros de Jean Philippe Toussaint -](http://www.leseditionsdeminuit.fr/livre-La_Cl%C3%A9_USB-3304-1-1-0-1.html), commencent à consacrer leur talent divinatoire à évaluer les conséquences économiques de cette (r)évolution. Dernier chiffre en date: 300 millions d'emplois au niveau mondial seraient concernés, d'après [Goldman Sachs](https://www.forbes.com/sites/jackkelly/2023/03/31/goldman-sachs-predicts-300-million-jobs-will-be-lost-or-degraded-by-artificial-intelligence/).

De manière plus pratique, la question de l'ouverture des corpus ayant servis à entraîner ces modèles ou de la licence à réutiliser des modèles pré-entraînés est importante à plusieurs égards. D'abord, dans une perspective scientifique, il apparaît compliqué d'évaluer la qualité d'un modèle ou lui proposer des alternatives sans accès à des données scientifiques de base comme le nombre de paramètres (inconnu pour GPT-4 par exemple). Mais, c'est aussi une question économique : si ces outils deviennent de plus en plus incontournables, quel sera le _business model_ de ce secteur ? Resteront-ils gratuits avec en contrepartie une réutilisation des données fournies, potentiellement opaque et difficilement contrôlable par l'utilisateur, ou ces services seront-ils monétisés ? 
```

> Cet [état de l'art](https://arxiv.org/pdf/2301.04655.pdf) publié en janvier est déjà largement dépassé.

Author: ```suggestion
```

d'[`OpenFlamingo`](https://laion.ai/blog/open-flamingo/) une version _open source_
de [Flamingo](https://www.deepmind.com/blog/tackling-multiple-tasks-with-a-single-visual-language-model),
le modèle de Google DeepMind.

Afin de pouvoir intégrer à la fois des fonctionnalités de reconnaissance d'image et textuelle, 
celui-ci s'appuie sur des composantes _open source_ pour les modèles de langage et 
de reconnaissance d'image et un jeu de données ouvert. Des exemples de réutilisation
en `Python` sont disponibles sur le [dépôt `Github`](https://github.com/mlfoundations/open_flamingo).
Dans le registre IA ouverte, une équipe de chercheurs de plusieurs universités américaines a
mis en oeuvre un `chatbot` ouvert, à partir des modèles `LLaMA` (Meta, voir [Newsletter #11](https://ssphub.netlify.app/infolettre/infolettre_11/))
ou `Alpaca` (Stanford) : [`Vicuna`](https://vicuna.lmsys.org/). 

`Databricks` s'appuie également sur `LLaMA` et `Alpaca` pour proposer 
son modèle de langage ouvert: [`Dolly`](https://www.databricks.com/blog/2023/03/24/hello-dolly-democratizing-magic-chatgpt-open-models.html).

> 

Author: @mpdebellefonjoubert j'ai fait une proposition juste au dessus, c'est ça que tu avais en tête ?

> Mozilla rejoint également le bal en investissant 30 millions de dollars pour lancer la _startup_ [Mozilla.ai](https://blog.mozilla.org/en/mozilla/introducing-mozilla-ai-investing-in-trustworthy-ai/). 

Author: ```suggestion
Mozilla rejoint également le bal en investissant 30 millions de dollars pour lancer la _startup_ [Mozilla.ai](https://blog.mozilla.org/en/mozilla/introducing-mozilla-ai-investing-in-trustworthy-ai/). Cette startup sert d'outil pour adopter la philosophie à l'origine du navigateur Mozilla au développement d'intelligences artificielles: développer des outils indépendants des principaux acteurs marchands du numérique, sur le modèle des communs plutôt que du bien privé. 
```

> Et la course folle n'est pas prête de s'arrêter puisque la prochaine étape 

Author: je vais plutôt proposer le truc 3D dans la prochaine newsletter, ce n'est plus pertinent compte tenu des derniers changements

des figures de la _tech_ (dont Elon Musk) et des universitaires réclame 
un moratoire dans le développement de nouveaux modèles d'intelligence
artificielle, dont les signataires soulignent à quel point il est difficile de les _"comprendre, prédire ou contrôler de manière fiable"_. Cette lettre a été rapidement critiquée, que ce soit par
[Andrew Ng](https://twitter.com/AndrewYNg/status/1641121451611947009?t=VyRYGWCjBeOxFOr0-bIJcQ&s=19)
ou par les
auteurs de l'article développant le concept de _"Stochastic Parrots"_ qui est cité dans la lettre 
([voir la réponse des auteurs](https://www.dair-institute.org/blog/letter-statement-March2023)).

## ... mais ne nous font pas oublier certaines autres nouveautés dignes d'intérêt

L'intensité de l'actualité autour de `ChatGPT` occulte beaucoup de
faits qui auraient, dans un autre contexte, amené à 
des discussions passionnées. 

Dans les faits majeurs quelques peu occultés par le bruit autour

> Ainsi, la [publication en open source de l'algorithme de recommandation de Twitter](https://www.lesechos.fr/tech-medias/medias/elon-musk-ouvre-une-partie-de-twitter-en-open-source-les-annonceurs-mefiants-1921111). 

Author: ```suggestion
Ainsi, la [publication en open source de l'algorithme de recommandation de Twitter](https://www.lesechos.fr/tech-medias/medias/elon-musk-ouvre-une-partie-de-twitter-en-open-source-les-annonceurs-mefiants-1921111)
```

Celui-ci permet de mieux comprendre la manière dont
fonctionne l'[algorithme de recommandation](https://www.theverge.com/2023/3/31/23664849/twitter-releases-algorithm-musk-open-source)
de `Twitter`, notamment la manière dont des [bulles de filtre](https://www.cnil.fr/fr/definition/bulle-de-filtre)
peuvent advenir après avoir cliqué sur un _post_ par curiosité. 

La publication de ce code prend place quelques jours après la découverte qu'une 
partie important du
code de `Twitter` était [déjà sur `Github`](https://www.theverge.com/2023/3/27/23657928/twitter-source-code-leak-github), sans
doute suite à une fuite après l'un des licenciements massifs des derniers mois. 

> `Docker` et `HuggingFace` nouent un [partenariat](https://www.docker.com/blog/docker-and-hugging-face-partner-to-democratize-ai/) pour faciliter la réutilisation

Reviewer: ```suggestion
Le XXX (_ce que fait Docker_)`Docker` noue un [partenariat](https://www.docker.com/blog/docker-and-hugging-face-partner-to-democratize-ai/) avec `HuggingFace`, qui rassemble une immense bibliothèque de modèles,  pour faciliter la réutilisation
```

Reviewer: Proposition de complément - à modifier/préciser - mais l'idée est toujours que la lettre soit accessible même à quelqu'un qui peut oublier entre deux lettres à quoi correspondent tous ces termes.

de modèles `HuggingFace`. Avec cette approche, il est maintenant possible de proposer
aux ré-utilisateurs, directement depuis `HuggingFace`, 
une application prête à l'emploi, avec tous

> les ingrédients pré-installés, pour

Reviewer: Ici je ne comprends pas bien le terme 'ingrédients', certes pas technique, mais du coup la valeur ajoutée ne me semble pas limpide. Peut-être que qu'une fois qu'il y aura quelques mots sur ce que fait docker cela sera plus clair.

tester ou enrichir un modèle. 

La plateforme `observable` propose
un [comparatif](https://observablehq.com/@observablehq/data-wrangling-translations) des principales syntaxes
de manipulation de données (`JavaScript`, `Python`, `R` et `SQL`). 

> Si vous appréciez les cartes, un _hashtag_ à suivre

Reviewer: ```suggestion
Enfin, si vous appréciez les cartes, un _hashtag_ à suivre
```

est le [#MapPromptMonday](https://twitter.com/hashtag/MapPromptMonday).

# Actualités du réseau: événements à venir

Place aux actualités de notre réseau avec les prochains événements
que nous organisons. 

> ## Première journée du réseau en avril (17 avril)

Reviewer: ```suggestion
## Première journée du réseau le 17 avril
```

Reviewer: Proposition pour qu'on imprime bien la date (mais si il y avait une raison de mettre en deux temps, comme tu préfères).

> une application prête à l'emploi, avec tous

Author: ```suggestion
une application prête à l'emploi
```

> les ingrédients pré-installés, pour

Author: ```suggestion
pour
```

est disponible en ligne [LIEN A VENIR]

## Replay de l'événement autour de l'OCRisation avec Christopher Kermorvant (29 mars)

[Christopher Kermorvant](https://www.linkedin.com/in/christopher-kermorvant-87158b2/?originalSubdomain=fr),
chercheur spécialisé en OCRisation et fondateur de [Teklia](https://teklia.com/)
a proposé au réseau une présentation très pédagogique sur l'extraction de texte 
avec des méthodes de _deep learning_.

A partir de l'exemple de recensements de la fin du XIXe siècle, nous avons
ainsi bénéficié d'une excellente introduction à l'histoire des techniques d'OCRisation
et la manière dont aujourd'hui ces modèles fonctionnent en associant reconnaissance d'image
et analyse textuelle. 

Le replay est [ici](https://ssphub.netlify.app/talk/ocrisation-teklia/) !

## Replay de la présentation de la documentation collaborative _Carpentries_ (28 mars)

Kate Burnett-Isaacs, de Statistics Canada, nous a présenté
l'initiative Meta Academy / Carpentries permettant de
construire une documentation francophone de référence
sur `R`, `Python` et `Git` pensée pour les utilisateurs de données.

> Le replay est [ici](https://ssphub.netlify.app/talk/ocrisation-teklia/) !

Author: erreur de lien, à changer

---
title: Infolettre Avril 2023, première quinzaine (#13)
#subtitle: 
toc: true
summary: |
  Infolettre du mois d'avril, première quinzaine
# Link this post with a project
#projects: [dataviz, recap2022]

# Date published
date: '2023-03-20T00:00:00Z'

# Date updated
lastmod: '2023-03-20T00:00:00Z'

# Is this an unpublished draft?
draft: false

# Show this page in the Featured widget?
featured: true

# Featured image
# Place an image named `featured.jpg/png` in this page's folder and customize its options here.
#image:
#  focal_point: ''
#  placement: 2
#  preview_only: false

authors:
  - linogaliana

tags:
#  - retrospective
#  - observable
#  - quarto
#  - deep learning
  - infolettre
  - chatGPT

categories:
#  - Insee
#  - Retrospective
  - Infolettre
---

{{% callout %}}

__*Vous désirez intégrer la liste de diffusion ? L'inscription se fait [ici](https://framaforms.org/integration-reseau-des-data-scientists-1676407156).*__

{{% /callout %}}

<br>

# Actualités de la data science

## Un foisonnement d'avancées... 

> L'actualité des modèles d'Intelligence Artificielle est très chargée ([résumé annonces du mois de mars](https://twitter.com/dustinvtran/status/1639159641635717120?t=tUqtzOHFvolL7MJef-M2Cw&s=19)) et cette lettre se concentrera sur les actualités majeures. 

Author: ```suggestion
L'actualité des modèles d'intelligence artificielle est très chargée ([résumé des annonces du mois de mars](https://twitter.com/dustinvtran/status/1639159641635717120?t=tUqtzOHFvolL7MJef-M2Cw&s=19)) et cette lettre se concentrera sur les actualités majeures. 
```

mis en oeuvre un `chatbot` ouvert, à partir des modèles `LLaMA` (Meta, voir [Newsletter #11](https://ssphub.netlify.app/infolettre/infolettre_11/))
ou `Alpaca` (Stanford) : [`Vicuna`](https://vicuna.lmsys.org/). Ce chatbot généraliste permet à un internaute de discuter sur une grande variété de sujets. L'avantage de celui-ci est néanmoins technique puisqu'il s'appuie sur des modèles de langage plus économes en ressources que les modèles type GPT-4. 

`Databricks` s'appuie également sur `LLaMA` et `Alpaca` pour proposer 
son modèle de langage ouvert: [`Dolly`](https://www.databricks.com/blog/2023/03/24/hello-dolly-democratizing-magic-chatgpt-open-models.html).

Mozilla rejoint également le bal en investissant 30 millions de dollars pour lancer la _startup_ [Mozilla.ai](https://blog.mozilla.org/en/mozilla/introducing-mozilla-ai-investing-in-trustworthy-ai/). Pour Mozilla, cette startup sert à adapter la philosophie à l'origine du navigateur Firefox au développement d'intelligences artificielles: proposer des outils ouverts indépendants des principaux acteurs marchands du numérique, sur le modèle des communs plutôt que du bien privé. 

> Et la course folle n'est pas prête de s'arrêter puisque la prochaine étape 

Author: ```suggestion
```

> est peut-être de générer des formes en 3D à partir [d'un jeu de données ouvertes en 3D](https://twitter.com/mattdeitke/status/1638608472525897728).

Author: ```suggestion
```

> {{< video src="3d.mp4" controls="yes" >}}

Author: ```suggestion
```

> _Source: [Twitter](https://twitter.com/mattdeitke/status/1638608472525897728)_

Author: ```suggestion
```
